src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -86,12 +86,12 @@ internal bool TryUnparse(StringBuilder sb)
         }
 
         /// <summary>
-        /// This class exposes <see cref="NgramExtractorTransform"/>/<see cref="NgramHashExtractingTransformer"/> arguments.
+        /// Advanced options for the <see cref="TextFeaturizingEstimator"/>.
         /// </summary>
-        internal sealed class Arguments : TransformInputBase
+        public sealed class Options : TransformInputBase
         {
             [Argument(ArgumentType.Required, HelpText = "New column definition (optional form: name:srcs).", Name = "Column", ShortName = "col", SortOrder = 1)]
-            public Column Columns;
+            internal Column Columns;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Dataset language or 'AutoDetect' to detect language per row.", ShortName = "lang", SortOrder = 3)]
             public Language Language = DefaultLanguage;
```

Comment:
>Language [](start = 28, length = 8)

Not related to your PR, but whole point of this thing is to set or autodetect language, we left autodetect outside of ML.NET so it's looks completely unnecessary here, but as I said in other comment, this is out of scope of this PR.

Reply:
Yes, let use this issue for tracking this [https://github.com/dotnet/machinelearning/issues/838](https://github.com/dotnet/machinelearning/issues/838).

---
In reply to: [264433503](https://github.com/dotnet/machinelearning/pull/2911#discussion_r264433503) [](ancestors = 264433503)